fixes.inc

```diff
@@ -205,6 +205,15 @@
  * AUTHOR:   im
  * AUTHOR:   Y_Less (https://github.com/Y-Less/)
  *
+ * FIX:      OnPlayerEnterVehicle_3
+ * PROBLEM:  No player animation when trying to enter the driver door of a locked vehicle
+ * SOLUTION: Leave the door unlocked and activate an animation when the player attemps
+ *           to enter a 'locked' vehicle.
+ * SEE:      "OnPlayerEnterVehicle".
+ * AUTHOR:   ziggi (https://github.com/ziggi)
+ * AUTHOR:   rt-2 (https://github.com/Y-Less/)
+ * POST:     https://github.com/ziggi/samp-enterfix
```

Comment:
I decided to keep only the forum post. No where else there was a github link as the original post, and the repo link is in ziggi's profile which is credited, and all other fixes had only one 'post'.
https://github.com/Open-GTO/sa-mp-fixes/pull/64/commits/c7a127cc66ca7fd9065c86092ebdf31cad02192c